e2e/tests/httproute-crud.spec.ts

```
@@ -0,0 +1,364 @@
+    const replaceInput = page.locator('.monaco-editor .replace-part input[aria-label="Replace"]');
+    await replaceInput.fill('/modified');
+    await page.locator('.monaco-editor .replace-part [aria-label*="Replace"]').first().click();
+    // Close find/replace
+    await page.keyboard.press('Escape');
+
+    // Switch back to form and verify the modified value
+    await page.locator('#create-type-radio-form').click();
+    await expect(page.getByText('/modified')).toBeVisible({ timeout: 10_000 });
+  });
+
+  test('deletes an HTTPRoute via the details actions menu', { tag: '@nightly' }, async ({ page }) => {
```

ESLint

[error] Replace `'deletes·an·HTTPRoute·via·the·details·actions·menu',·{·tag:·'`@nightly`'·},` with `⏎····'deletes·an·HTTPRoute·via·the·details·actions·menu',⏎····{·tag:·'`@nightly`'·},⏎···`

(prettier/prettier)